csc/templates/csc-pvc.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.mountpoint }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ .Release.Name }}-pvc
+  {{- if .Values.mountpoint.gid }}
+  annotations:
+    pv.beta.kubernetes.io/gid: "{{ .Values.mountpoint.gid }}"
```

Comment:
I think there might also be a uid one you can use here too, if you end up having problems.  I just like debugging it as I go, as you say

Reply:
Sounds like a good idea, so I'll add it.